src/main/kotlin/de/bund/digitalservice/useid/config/CustomWebClientExchangeTagsProvider.kt

```diff
@@ -0,0 +1,35 @@
+package de.bund.digitalservice.useid.config
+
+import de.bund.digitalservice.useid.tenant.PARAM_NAME_TENANT_ID
+import de.bund.digitalservice.useid.tenant.REQUEST_ATTR_TENANT
+import de.bund.digitalservice.useid.tenant.Tenant
+import io.micrometer.core.instrument.Tag
+import jakarta.servlet.http.HttpServletRequest
+import jakarta.servlet.http.HttpServletResponse
+import org.springframework.boot.actuate.metrics.web.servlet.WebMvcTagsContributor
+import org.springframework.stereotype.Component
+
+@Component
+class CustomWebClientExchangeTagsProvider : WebMvcTagsContributor {
```

Comment:
Could we add a comment here what this class does?

Reply:
This was Patricks work. I added a comment with my knowledge. Feel free to edit and expand @phuesler